jobs/config.json

```diff
@@ -6726,6 +6726,26 @@
       "sig-cluster-lifecycle"
     ]
   },
+  "ci-kubernetes-e2e-gke-device-plugin-gpu": {
+    "args": [
+      "--check-leaked-resources",
+      "--deployment=gke",
+      "--extract=ci/latest",
+      "--gcp-cloud-sdk=gs://cloud-sdk-testing/ci/staging",
+      "--gcp-node-image=gci",
+      "--gcp-project=k8s-jkns-e2e-gke-gpus",
```

Comment:
Can you create a new project instead of using the existing one. You will have to request GPU quota for the new project.